src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -395,9 +418,9 @@ const MapView: React.FC<MapViewProps> = ({
             let imageSize = 0;
 
             const zoomLevel = map.getZoom();
-            if (zoomLevel > 17 && zoomLevel < 20) {
+            if (zoomLevel > view + 1 && zoomLevel < STREET_ZOOM) {
```

Comment:
Is zoom level not automatically set to zoomLevel?

Reply:
The default zoom level is view after clicking on a neighborhood/street zoom after searching for a place, then a user might zoom in or out. There's no broader local constant for zoom level.